torch/jit/_recursive.py

```diff
@@ -575,6 +598,16 @@ def init_fn(script_module):
         for name, cpp_module in torch._C.ModuleDict(script_module._c).items():
             setattr(script_module, name, wrap_cpp_module(cpp_module))
         script_module._concrete_type = torch._C.ConcreteModuleType.from_jit_type(script_module._c._type())
+        idx = 0
+        for fn_name in script_module._concrete_type.get_forward_pre_hooks():
```

Comment:
nit: get rid of manual `idx` incrementing and use `for idx, fn_name in enumerate(script_module._concrete_type.get_forward_pre_hooks()):`